src/templates/ObservabilityPackDetails.js

```diff
@@ -331,6 +307,15 @@ export const pageQuery = graphql`
         screenshots
         url
       }
+      alerts {
```

Comment:
alerts, synthetics, and nerdpacks are not part of the API yet, so we probably want to remove these for now.

Reply:
removed